pkg/interactive/addon.go

```diff
@@ -0,0 +1,67 @@
+package interactive
+
+import (
+	"fmt"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"net"
+	"strconv"
+)
+
+func GetAddonParameter(addOnInstallationParam *cmv1.AddOnInstallationParameter, param *cmv1.AddOnParameter, input Input) (string, error) {
+	// Set default value based on existing parameter, otherwise use parameter default
+	dflt := param.DefaultValue()
+	if addOnInstallationParam != nil {
+		dflt = addOnInstallationParam.Value()
+	}
```

Comment:
since we're only using `addOnInstallationParam` to override the default value when editing an addon installation, perhaps we could instead pass the default from the caller?